samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.1</TargetFrameworks>
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
```

Comment:
I hesitate to remove this because confirming everything works in command-line and VS runs as well as our functional tests might miss something. Have at least confirmed the Microsoft.NET.Sdk.Web targets do *not* contain anything similar.

Anyone know if these lines in the web site project files can be removed?